examples/kafka-cluster/src/test/java/com/example/kafkacluster/KafkaContainerCluster.java

```diff
@@ -60,6 +60,7 @@ public KafkaContainerCluster(String confluentPlatformVersion, int brokersNum, in
                         .dependsOn(this.zookeeper)
                         .withExternalZookeeper("zookeeper:" + KafkaContainer.ZOOKEEPER_PORT)
                         .withEnv("KAFKA_BROKER_ID", brokerNum + "")
+                        .withEnv("KAFKA_NODE_ID", brokerNum + "")
```

Comment:
```suggestion
```

With the way the examples are structured now, I don't think this is needed anymore, right?

Reply:
Node.is is mandatory for kraft however the lack of this property is covered by https://github.com/testcontainers/testcontainers-java/blob/bba06bde0a4b7fc232c3fa3c7e64607021af04f8/modules/kafka/src/main/java/org/testcontainers/containers/KafkaContainer.java#L139-L143

Reply:
But this example class is not used for kraft mode, correct? Hence, we don't need this change either way?

Reply:
this is a leftover from previous commits.